ios/OTSessionManager.swift

```diff
@@ -18,6 +18,14 @@ class OTSessionManager: RCTEventEmitter {
   var publisherPreface: String = "publisher:";
   var subscriberPreface: String = "subscriber:";
 
+  deinit {
+    OTRN.sharedState.subscriberStreams.removeAll();
+    OTRN.sharedState.session = nil;
+    OTRN.sharedState.isPublishing.removeAll();
+    OTRN.sharedState.publishers.removeAll();
+    OTRN.sharedState.subscribers.removeAll();
+  }
```

Comment:
Should it all be set to `nil`? Just raise a question to think.

Reply:
@Lucashuang0802 can't set the `subscriberStreams`, `isPublishing`, `publishers`, or `subscribers` to nil because of the types on the dictionaries